framework/core/events_test.ts

```diff
@@ -0,0 +1,687 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+import {
+    assert,
+    assertEquals,
+    assertThrows,
+    fail,
+} from "std/testing/asserts.ts"
+import { EventEmitter, on, once, WrappedFunction } from "./events.ts"
+
+const shouldNeverBeEmitted = () => {
+    fail("Should never be called")
+}
+
+Deno.test({
+    name:
+        'fw/core/events When adding a new event, "eventListener" event is fired before adding the listener',
```

Comment:
I prefixed the name/description of a ll tests with a reference to the file under test as has been done in other tests. That was the only modification I made to this file.